test/units/testsuite-35.sh

```diff
@@ -462,6 +462,28 @@ EOF
     assert_eq "$(systemctl --property SubState --value show user@"$(id -u logind-test-user)".service)" "dead"
 }
 
+test_session_properties() {
+    local s
+
+    if [[ ! -c /dev/tty2 ]]; then
+        echo "/dev/tty2 does not exist, skipping test ${FUNCNAME[0]}."
+        return
+    fi
+
+    trap cleanup_session RETURN
+    create_session
+
+    s=$(busctl call org.freedesktop.login1 /org/freedesktop/login1 org.freedesktop.login1.Manager ListSessions |\
+        awk '{ if ($5 == "\"logind-test-user\"") print $7 }' | tr -d \")
```

Comment:
Does this work if there exist multiple sessions??
I think it is safer to build session object path from the output of `loginctl`:
```
s = $(loginctl --no-legend | grep logind-test-user | awk '{ print $1 }')
/usr/lib/systemd/tests/manual/test-session-properties "/org/freedesktop/login1/session/_3${s?}"
```

Reply:
If it's reliable to construct the object path like this, why not... But it's even possible that there be more than one session, unless there's a bug in the test code?

Reply:
Anyway, I changed this.